sdk/src/crypto/time_stamp/http_request.rs

```diff
@@ -55,11 +56,16 @@ pub fn default_rfc3161_request(
     let mut local_log = StatusTracker::default();
     let ctp = CertificateTrustPolicy::passthrough();
 
+    // TO REVIEW: I think in this case, we're doing structural validation of
+    // time stamps but not trust validation. If so, the below is correct. (I think.)
```

Comment:
TODO: internal

Reply:
Yes, following the code path we explicitly set `CertificateTrustPolicy::passthrough` which stops before trust checks occur.  A bigger question is if we need settings here at all versus passing `verify_trust` through the `passthrough` field of `CertificateTrustPolicy`.